README.md

```diff
@@ -24,20 +24,23 @@ pre-commit install
 
 The simplest way of executing the server is:
 ```
-sudo -i python qibosoc/src/qibosoc/rfsoc_server.py
-``` 
+sudo -i python ABSOLUTE_PATH/qibosoq/src/qibosoq/rfsoc_server.py
```

Comment:
```suggestion
sudo -i python /absolute/path/to/qibosoq/src/qibosoq/rfsoc_server.py
```
It's a placeholder anyhow, but it looks closer to the actual result.

Another common option is:
```suggestion
sudo -i python <absolute-path-to-qibosoq>/src/qibosoq/rfsoc_server.py
```

Reply:
(in principle even upper case it's fine, and you should be a bit dumb to believe it to be an actual folder named `ABSOLUTE_PATH`, but it might look like a bash variable, just missing a `$` sign... even though reading the name is pretty clear, but it is always hard to infer what is clear and what is not for someone else)

Reply:
I like the second option, thanks for the suggestion!